subsys/bluetooth/audio/bap_stream.c

```diff
@@ -277,8 +284,7 @@ int bt_bap_stream_send(struct bt_bap_stream *stream, struct net_buf *buf,
 
 	/* TODO: Add checks for broadcast sink */
 
-	return bt_iso_chan_send(bt_bap_stream_iso_chan_get(stream),
-				buf, seq_num, ts);
+	return 0;
```

Comment:
Since `err` (or `ret`) returns the number of bytes sent, this should be
```suggestion
	return err;
```